java/edit-feature-attachments/README.md

```diff
@@ -1,24 +1,41 @@
 # Edit feature attachments
-Add, delete or fetch attachments for a specific feature in a feature layer.
 
-![Edit Feature Attachments](edit-feature-attachments.png)
+Add, delete, and download attachments for features from a service.
+
+![Image of edit feature attachments](edit-feature-attachments.png)
+*****************
```

Comment:
not sure if this is markdown or not, but guide says a newline should appear before and after each heading, and this butts up against the Use case heading.

Reply:
oops! I've removed it now